app/src/main/java/org/schabi/newpipe/player/BasePlayer.java

```diff
@@ -954,12 +954,19 @@ public void onPlayPause() {
 
     public void onFastRewind() {
         if (DEBUG) Log.d(TAG, "onFastRewind() called");
-        seekBy(-FAST_FORWARD_REWIND_AMOUNT_MILLIS);
+        seekBy(-getSeekDuration());
     }
 
     public void onFastForward() {
         if (DEBUG) Log.d(TAG, "onFastForward() called");
-        seekBy(FAST_FORWARD_REWIND_AMOUNT_MILLIS);
+        seekBy(getSeekDuration());
+    }
+
+    private int getSeekDuration() {
+        final SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
+        final String key = context.getString(R.string.seek_duration_key);
+        final String value = prefs.getString(key, FAST_FORWARD_REWIND_DEFAULT_AMOUNT_MILLIS);
```

Comment:
You could remove `FAST_FORWARD_REWIND_DEFAULT_AMOUNT_MILLIS`, since it is a duplicate of the `seek_duration_default_value` string.
```suggestion
        final String value = prefs.getString(key, context.getString(R.string.seek_duration_default_value));
```